File: papi/lista1/exercicio4.py
numero = 0

while True:
    try:
        numero = int(input("Digite un número: "))
        break
    except ValueError:
        print("Por favor digite um número")

print("O antecessor de", numero, "é", numero - 1)
print("O sucessor de", numero, "é", numero + 1)

File: papi/lista1/exercicio4.kt
package lista1

fun main() {
    var numero: Int
    while (true) {
        try {
            print("Por favor informe um número inteiro: ")
            val input = readLine()
            numero = input!!.toInt()
            break
        } catch (e: NumberFormatException) {
            println("Por favor digite um número inteiro")
        }
    }

    print("O sucessor do número é: ")
    println(numero + 1)
    print("O antecessor do número é: ")
    println(numero - 1)
}